Servers/models/EU/answerEU.model.ts

```
@@ -0,0 +1,63 @@
+import { Column, DataType, ForeignKey, Model, Table } from "sequelize-typescript";
+import { QuestionStructEUModel } from "./questionStructEU.model";
+import { AssessmentEUModel } from "./assessmentEU.model";
+
+export type AnswerEU = {
+  id?: number; // auto generated by database
+  question_id: number; // gets assigned from the structure
+  answer?: string; // won't get any values, will be filled by user
+  created_at?: Date;
+  assessment_id: number;
+  dropdown_options?: any[]; // gets assigned from the structure
+  evidence_files?: Object[]; // gets assigned from the structure
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)